### ISSUE TITLE
CI: broken links for https://some-random-api.ml/facts/dog

### ISSUE THREAD

**Issue by vdice** [fermyon/developer#590]
CI is currently failing due to broken links for the following url: https://some-random-api.ml/facts/dog

```
Finished! 5050 links found. 4603 excluded. 3 broken.
Elapsed time: 1 minute, 5 seconds
Some links are broken, retrying to check for transient errors
The list of broken links are

     1	├─BROKEN─ https://some-random-api.ml/facts/dog (ERRNO_ENOTFOUND)
     2	├─BROKEN─ https://some-random-api.ml/facts/dog (HTTP_undefined)
     3	├─BROKEN─ https://some-random-api.ml/facts/dog (HTTP_undefined)
```

Example: https://github.com/fermyon/developer/actions/runs/4892883847/jobs/8735171302?pr=588

Indeed, I'm seeing the same behavior currently:
```
$ curl https://some-random-api.ml/facts/dog
curl: (28) Failed to connect to some-random-api.ml port 443 after 75472 ms: Couldn't connect to server
```

**Comment by itowlson** [comment]
@karthik2804 @rajatjindal You were working on a random facts API last week I think?  Is that ready for us to swap over the docs?

**Comment by karthik2804** [comment]
@itowlson  I have not gotten around to it yet. Hoping to get to it tonight. My apologies for the delay.

**Comment by tpmccallum** [comment]
I just did a quick grep on the developer documentation repo so we can go ahead and update the necessary files; when the time/fix arrives.

```
$ grep -rl "https://some-random-api" .        
./content/spin/javascript-components.md
./content/spin/go-components.md
./content/spin/python-components.md
./content/spin/rust-components.md
./content/spin/http-outbound.md
./static/data.json
```

**Comment by tpmccallum** [comment]
Fixed bt #628
